yarn.lock

```diff
@@ -1174,7 +1174,6 @@
 "@emotion/cache@^10.0.15":
   version "10.0.15"
   resolved "https://registry.yarnpkg.com/@emotion/cache/-/cache-10.0.15.tgz#b81767b48015aae2689c60373992145c67b8de02"
-  integrity sha512-8VthgeKhlGeTXSW1JN7I14AnAaiFPbOrqNqg3dPoGCZ3bnMjkrmRU0zrx0BtBw9esBaPaQgDB9y0tVgAGT2Mrg==
```

Comment:
@BJHunnicutt can you upgrade your version of yarn - the integrity checks are being stripped from the yarn.lock file.

Once we resolve #728, we shouldn't have to worry about this anymore. We can check in a version of yarn using yarn-path as described here to avoid this issue in the future. More info: yarnpkg/yarn#6440